### DISCUSSION TITLE
how can I get the url of the python jupyter server so I can access it from the browser too

### DISCUSSION THREAD

**Discussion by chanansh** [microsoft/vscode-jupyter#14394] [Questions and Answers]
I am using the built-in kernel python for the jupyter notebook. 
How can I get the url (port and token) to access the same notebook server from a browser?
The kernel is the local venv 
![image](https://github.com/microsoft/vscode-jupyter/assets/5981644/9eb8a8f8-5a2c-4024-b497-ba731ceb88cc)
But I would like to go to localhost:8888?token=xxx to be able to see it in the browser. Could not find a way to get it.

**Comment by DonJayamanne** [comment]
> How can I get the url (port and token) to access the same notebook server from a browser?
The kernel is the local venv

Unfortunately there is none
We do not start a Jupyter server when running kernels locally

out of curiosity why do you need to access this? Ie what functionally are you looking for that isn’t available in vscode?

**Comment by chanansh** [comment]
Jupyter add-ons such as RISE (presentation notebook as deck) do not work in vscode. I would like to be able to access via browser for these functionality. Also fo seeing how will the notebook look like in GitHub

Thanks,
HS
________________________________
